### ISSUE TITLE
nginx doesn't open all ports from the range in the 1.23.3 version

### ISSUE THREAD

**Issue by paivin-dn** [nginxinc/docker-nginx#744]
In previous versions(<=1.23.2) I was using the following config to open the port range:
```
worker_processes auto;

user nobody nogroup;
error_log  /var/log/nginx/error.log warn;
pid /var/run/nginx.pid;

events {
  worker_connections 2048;
  accept_mutex off;
}
stream {
  resolver 127.0.0.11 ipv6=off valid=30s;

  upstream my_server {
    server <ip>:2101;
  }

  server {
    listen 2101;
    proxy_pass my_server;
    proxy_connect_timeout 5s;
  }

  server {
    listen 3100-3150;
    set $my_server <ip>:$server_port;
    proxy_pass $my_server;
    proxy_connect_timeout 5s;
  }
}
```
But after updating to the 1.23.3 version I found that nginx opens only the subset of ports:
```bash
netstat -tulpan
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp        0      0 0.0.0.0:2101            0.0.0.0:*               LISTEN      1/nginx: master pro
tcp        0      0 0.0.0.0:3100            0.0.0.0:*               LISTEN      1/nginx: master pro
```

**Comment by paivin-dn** [comment]
@thresheek thank you! Let's wait for release 1.23.4

**Comment by thresheek** [comment]
Fixed now with 1.23.4 images.
